Frontend/src/components/salon-components/salonServices.tsx

```
@@ -7,26 +7,31 @@ import { useStyles } from "../../app/salon/bookings/style/styles";
 import {
   useBookingActions,
 } from "@/providers/booking-provider";
-import { useEmployeeTechnicianActions } from "@/providers/employeeTechnician-provider";
 import "@ant-design/v5-patch-for-react-19";
 import { ISalonService } from "@/providers/salonService-provider/context";
 import { useSalonServiceActions, useSalonServiceState } from "@/providers/salonService-provider";
 
 
 
-const SalonServices = ({ salonServices: passedSalonServices }: { salonServices?: ISalonService[] }) => {
+const SalonServices = ({}: { salonServices?: ISalonService[] }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)